### PR TITLE
New scientist traitor item: Australian Slime Mutator / Spider Injector

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
add: New scientist traitor item: Australian Slime Mutator / Spider Injector, use it on a gold slime extract to create 3 neutral broodmother spiders, make them sentient and start your own hive.
/:cl:

'ello mates, Me and my syndicate expedition team have recently returned from my journey to the Australicus sector and crikey the spiders are big there. Fucking the size of a bear. Luckily one of my fellow expeditioners managed to knock one of the fuckers out with a boomerang and we took 'er to our labs. We managed to extract some of their extract which is known to create tame offspring when injected into a gold slime core.

However, if you give it sentience and tell it to do whatever you want, maybe you can use it for a useful purpose?

10 TC item, lets you inject a gold-slime core for some midwife/broodmother spiders that can help you start a spider army. Price can be raised if people think 10 is too little. It spawns 3 instead of 1 to keep consistent, but it can be lowered to 1 spider.

### CODE REVIEW

**Comment by knacker48** [comment]
Good to see that spider armies still exist in some form, even if I can no longer send an unending swarm at blobs or xenomorphs to protect the station

**Comment by FantasticFwoosh** [comment]
``/datum/chemical_reaction/slime/slimemobspawn/spider
	name = "Slime **Crit** Traitor Spider"``

Typo here for slime *Kit, what' a slime crit? Otherwise a nice PR but golds are very far away for three broods by the time a traitor round is over depending on how hard you powergame xenobio.

**Comment by Qustinnus** [comment]
critter

**Comment by Arathianen** [comment]
This is a big brained PR for big brained and very sexy people.

**Comment by knacker48** [comment]
@FantasticFwoosh Well as someone who plays xenobio a lot I can get golds around the half hour mark, maybe earlier with some luck, which I think is early enough. That said its not golds you need its also light pinks so that adds the to setup time

**Comment by Qustinnus** [comment]
it's also pretty strong to have a xeno army

**Comment by Arathianen** [comment]
@FantasticFwoosh It's golds AND light pinks.

Spider armies aren't really supposed to be a powergame option. They are a fun gimmick and get you killed also 90% of the time.

**Comment by XDTM** [comment]
It's nice to have a way to get spiders, but it always makes me sad when a whole mechanic is restricted to traitor mode because it's in the uplink

**Comment by FantasticFwoosh** [comment]
> @FantasticFwoosh It's golds AND light pinks.

``name = "gold slime extract"
activate_reagents = list(/datum/reagent/blood,/datum/reagent/toxin/plasma,/datum/reagent/water, /datum/reagent/spider_extract)``

That isn't exactly true sorry, everything you need to know you can find [here](https://github.com/tgstation/tgstation/pull/44559/files#diff-ec1884d654032f003db2d6e6b67567f3).

**Comment by Qustinnus** [comment]
it was removed from gold cores a while ago due to it being to griefy sadly.

**Comment by Arathianen** [comment]
@FantasticFwoosh I should clarify. It works on golds, but you also need light pinks to actually make them sentient. Otherwise, they are brain dead and completely useless.

**Comment by Qustinnus** [comment]
@FantasticFwoosh you're misinterpreting what he's saying. you need a sentience potion from a pink slime to make the spiders useful.

**Comment by Tlaltecuhtli** [comment]
why would the syndicate know about super secret xenobio slime tech? make it so it turns spiders into midwifes

**Comment by Qustinnus** [comment]
why would syndicates know about half of their items?

**Comment by Hathkar** [comment]
10 TC seems extremely low for an item comparable to or arguably stronger than romerol. Considering you can give the spiders directives, they're near infinite in spawning once you have a single corpse to feed them.

**Comment by ArcaneMusic** [comment]
You guys see this as spiders coming back.

I see this as an excuse for more _S P A C E  A U S T R A L I A_

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1728,3 +1728,11 @@
...
 	if(method==PATCH || method==INGEST || method==INJECT || (method == VAPOR && prob(min(reac_volume,100)*(1 - touch_protection))))
 		L.ForceContractDisease(new /datum/disease/transformation/gondola(), FALSE, TRUE)
+
+
+/datum/reagent/spider_extract
+	name = "Spider Extract"
+	description = "A highly specialized extract coming from the Australicus sector, used to create broodmother spiders."
+	color = "#ED2939"
+	taste_description = "upside down"
+	can_synth = FALSE
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: awwwwww

C'mon, let a joint operation between botany, xenobiology, and Lady Luck let people make some non-antag broodmother spiders (without the involvement of a traitor) once in a blue moon.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1632,6 +1632,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+
+/datum/uplink_item/role_restricted/spider_injector
+	name = "Australian Slime Mutator"
+	desc = "Crikey mate, it's been a wild travel from the Australicus sector but we've managed to get \
+			some special spider extract from the giant spiders down there. Use this injector on a gold slime core \
+			to create a few of the same type of spiders we found on the planets over there. They're a bit tame until you \
+			also give them a bit of sentience though."
+	item = /obj/item/reagent_containers/syringe/spider_extract
+	cost = 10
+	restricted_roles = list("Research Director", "Scientist", "Roboticist")
```
Comment (ATH1909) [inline code/modules/uplink/uplink_items.dm]: It hurts me to say this as a roboticist main, but roboticists shouldn't be able to buy this item; they only get xenobio access on skeleton shifts (or if they hack in there). To me, it's sort of like how medical doctors can't buy magillitis like a geneticist or a CMO can.

**Comment by Arathianen** [comment]
@Hathkar I think the main difference is that the spider injector by itself is useless, while romerol can be used instantly.

Spider injector needs light pink slime and golden slime to actually be useful.

**Comment by Shadowflame909** [comment]
Fastest I've seen tier 4s be speed runned is 45 minutes.

The RnG really does keep it balanced.

Especially since you need a tier 3 of another route altogether.

Now especially on bagil, I think the scientist who buys this item is going to end up having the shuttle called before he can use it.

**Comment by optimumtact** [comment]
I really hate australians

**Comment by PKPenguin321** [comment]
reported to github for racism

**Comment by Arathianen** [comment]
Oranges confirmed racist.

I thought we were a progressive server here.

**Comment by YakumoChen** [comment]
> 
> 
> Fastest I've seen tier 4s be speed runned is 45 minutes.
> 
> The RnG really does keep it balanced.
> 
> Especially since you need a tier 3 of another route altogether.
> 
> Now especially on bagil, I think the scientist who buys this item is going to end up having the shuttle called before he can use it.

just wait to buy it until you need it?? Good traitors don't spend all their TC at roundstart.

**Comment by necromanceranne** [comment]
Time for Australicus language.

**Comment by fludd12** [comment]
Please make sure that, if the chemical is to be used on a gold extract, it is somehow blacklisted from the options that self-sustaining crossbreeds provide - otherwise, self-sustaining crossbreeds will be able to produce this chemical, meaning they will be accessible to non-traitors.

**Comment by MCterra10** [comment]
@optimumtact you bloody New Zealanders

**Comment by Qustinnus** [comment]
@fludd12 already done

**Comment by bigfatbananacyclops** [comment]
i feel like people will constantly do this as a scientist and eventually everyone will get tired of it. Like a 10 tc romerol

**Comment by habader** [comment]
> i feel like people will constantly do this as a scientist and eventually everyone will get tired of it. Like a 10 tc romerol

its barely any different from the old way to get spiders

**Comment by Arathianen** [comment]
> > i feel like people will constantly do this as a scientist and eventually everyone will get tired of it. Like a 10 tc romerol
> 
> its barely any different from the old way to get spiders

It's exactly the same, in fact. Just instead of plasma, you need to put this special extract.

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -1632,6 +1632,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 15
 	restricted_roles = list("Clown")
 
+
+/datum/uplink_item/role_restricted/spider_injector
+	name = "Australian Slime Mutator"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	name = "Australicus Slime Mutator"
```

Let's not break character!!!!

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/uplink/uplink_items.dm]: this can only be used if you do xenobio tho

**Comment by Tlaltecuhtli** [comment]
meta generates counter meta, wait a few weeks and you ll have ais pre emtively syphoning xenobio as they see a cobweb

**Pull request merged**